### PR TITLE
py/bitbox02: spell out the optional dependency on rlp for ethereum

### PR DESCRIPTION
**Pull request by x1ddos**
To use this locally:

    pip install -e .[ethereum]

Users who want to install a released version can do:

    pip install bitbox02[ethereum]

See https://www.python.org/dev/peps/pep-0508/#environment-markers
for docs on how package dependencies can be specified.

### CODE REVIEW

**Comment by x1ddos** [comment]
Just trying to please check-pep8

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 ## [Unreleased]
 - RestoreFrommnemonic: ported to Rust. Will now return UserAbortError on user abort instead of GenericError.
+- Added [rlp] as an optional dependency of the bitbox02 Python package using an environment marker
```
Comment (benma) [inline CHANGELOG.md:5]: maybe there should be  a separate changelog for the python package? after all it does not follow the same release schedule as the firmware, and has a different version, etc.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
...
+  `ethereum`; see PEP-508 for environment markers
+
+[rlp]: https://pypi.org/project/rlp/
```
Comment (benma) [inline CHANGELOG.md:8]: what does this do?

**Review by benma — APPROVED** [review]

**Comment by NickeZ** [comment]
You probably also need to update some README in the `py` directory.

**Review by x1ddos — COMMENTED** [review]
> You probably also need to update some README in the `py` directory.

done

made some more changes and added `all` optional extras so PTAL

**Pull request closed without merging**